codalab/worker/local_run/local_run_manager.py

```diff
@@ -204,15 +204,18 @@ def create_run(self, bundle, resources):
             bundle=bundle,
             bundle_path=os.path.realpath(bundle_path),
             resources=resources,
-            start_time=now,
+            bundle_start_time=now,
+            container_start_time=None,
+            container_time_total=0,
```

Comment:
Semantically, shouldn't these be `None`?

Reply:
This state object generally acts as a source of information as of the last time the run manager checked the run. My reasoning with setting the runtime variables to 0 was that at the moment of creation the bundle has not spent any time yet, and the container runtime stays at 0 until the container starts, which makes downstream cases a lot simpler (not having to deal with None checking etc.)